main.go

```diff
@@ -38,27 +34,25 @@ func packageWorker(ctx context.Context, workChan <-chan library.Package, outputC
 }
 
 func main() {
+	ctx := context.Background()
 	cli, err := client.NewEnvClient()
 	if err != nil {
 		log.Fatalf("error creating docker client: %v", err)
 	}
 
-	sess := session.Must(session.NewSession(&aws.Config{
-		Region: aws.String("us-east-1")}))
-
-	cs, err := consumer.NewSQSConsumer(sess, os.Getenv("OSSMALWARE_QUEUE_NAME"))
+	queueURL := os.Getenv("OSSMALWARE_QUEUE_URL")
+	cs, err := consumer.NewPubSubConsumer(ctx, queueURL)
 	if err != nil {
 		log.Fatalf("error creating consumer: %v", err)
 	}
 
-	uploader, err := uploader.NewS3Uploader(sess, os.Getenv("OSSMALWARE_UPLOAD_BUCKET"))
+	uploader, err := uploader.NewBlobUploader(ctx, os.Getenv("OSSMALWARE_UPLOAD_BUCKET_URL")s)
```

Comment:
```suggestion
	uploadURL := os.Getenv("OSSMALWARE_UPLOAD_BUCKET_URL")
	uploader, err := uploader.NewBlobUploader(ctx, uploadURL)
```

Minor syntax error here, and we might as well make it have the same structure as `queueURL` above.

Reply:
Whoops, fixed!